lib/widgets/breez_avatar_dialog.dart

```diff
@@ -11,147 +12,180 @@ import 'package:image/image.dart' as DartImage;
 import 'package:image_cropper/image_cropper.dart';
 import 'package:image_picker/image_picker.dart';
 
+import 'flushbar.dart';
+
 int scaledWidth = 200;
 var _transparentImage = DartImage.Image(scaledWidth, scaledWidth);
 
 Widget breezAvatarDialog(BuildContext context, UserProfileBloc userBloc) {
   AutoSizeGroup _autoSizeGroup = AutoSizeGroup();
   BreezUserModel _currentSettings;
+  List<int> _imageData;
 
   final _nameInputController = TextEditingController();
   userBloc.userPreviewStream.listen((user) {
     _nameInputController.text = user.name;
     _currentSettings = user;
   });
 
-  Future _pickImage(BuildContext context) async {
+  Future _pickImage(BuildContext context, setState) async {
     return ImagePicker.pickImage(source: ImageSource.gallery).then((file) {
       ImageCropper.cropImage(
         sourcePath: file.path,
         aspectRatio: CropAspectRatio(ratioX: 1.0, ratioY: 1.0),
       ).then((file) {
         if (file != null) {
           file.readAsBytes().then(scaleAndFormatPNG).then((image) {
```

Comment:
Instead of reading the file to memory isn't it better to use the file url instead?
This way we can use this url in the BreezAvatar instead of adding an extra "imageData" parameter.

Reply:
[Returned the file and passed the url to BreezAvatar, it's bytes to UploadProfilePicture user action](https://github.com/breez/breezmobile/pull/283/commits/890899a2ba0a5eae80bfe2c3d5f5391d43b6cf25)